packages/sit-onyx/src/components/OnyxNavAppArea/OnyxNavAppArea.vue

```diff
@@ -61,7 +61,7 @@ const buttonLabel = computed(() => props.label ?? t.value("navigation.goToHome")
     padding: var(--onyx-spacing-md);
     font-weight: 600;
     white-space: pre-line;
-    max-height: 100%;
+    max-height: 100cqh;
```

Comment:
Fancy^^ Never saw this unit before :D
A comment might help if we take a look at this in the future or we just set the logo max width also to 1.5rem

Reply:
added comment